include/node.tcc

```diff
@@ -128,6 +130,49 @@ bool mpm::Node<Tdim, Tdof, Tnphases>::update_internal_force(
   return status;
 }
 
+//! Update mixture internal force (body force / traction force)
```

Comment:
Isn't it better to include all the specific 2 phase nodal functions to a separate .tcc file?